cvat/apps/dataset_manager/bindings.py

```diff
@@ -490,17 +541,36 @@ def _make_image(i, **kwargs):
                 'path': frame_data.name + ext,
                 'size': (frame_data.height, frame_data.width),
             }
-            if include_images:
+
+            if dimension == DimensionType.DIM_3D:
+                dm_image = _make_image(frame_data.id, **image_args)
+            elif include_images:
                 dm_image = _make_image(frame_data.idx, **image_args)
             else:
                 dm_image = Image(**image_args)
             dm_anno = self._read_cvat_anno(frame_data, task_data)
-            dm_item = datumaro.DatasetItem(id=osp.splitext(frame_data.name)[0],
-                annotations=dm_anno, image=dm_image,
-                attributes={'frame': frame_data.frame})
+
+            if dimension == DimensionType.DIM_2D:
+                dm_item = datumaro.DatasetItem(id=osp.splitext(frame_data.name)[0],
+                                               annotations=dm_anno, image=dm_image,
+                                               attributes={'frame': frame_data.frame})
+            elif dimension == DimensionType.DIM_3D:
+                attributes = {'frame': frame_data.frame}
+                if format_type == "point_cloud":
+                    attributes["name"] = self._user["name"]
+                    attributes["createdAt"] = self._user["createdAt"]
+                    attributes["updatedAt"] = self._user["updatedAt"]
+                    attributes["labels"] = []
+                    for label in frame_data.labels.values():
+                        attributes["labels"].append({"label_id": label.id, "name": label.name, "color": label.color})
```

Comment:
1. If color information is not used in this format, there is no point in passing and storing it.
2. If it is needed and is not item-specific, it must be passed in categories (`MaskCategories`).

Reply:
The color is specific to this attribute, required to be consistent for uploading in other apps